src/views/layout.njk

```diff
@@ -42,22 +28,29 @@
   {% block download %}{% endblock %}
 {% endblock %}
 
+
+
+{% block pageTitle %}
```

Comment:
I do not think it is needed. Layout is for extension so usually extending pages will declare it.